doc/articles/concepts/overview/why-uno-platform.md

```diff
@@ -0,0 +1,276 @@
+
+## Localization included
+
+Uno Platform supports localization from the `x:Uid` feature in XAML, as well as using resource files from `ResourceManager` in plain C# code. Localization is done once for all your target platforms.
+
+## XAML Trimming
+
+XAML Trimming is a feature that allows for trimming the size of WebAssembly apps down by removing XAML styles that are not used in your app, helping to bring down the size of your application.
+
+## Lightweight styling
+
+Lightweight styling is a way to customize the appearance of XAML controls by overriding their default brushes, fonts, and numeric properties. Lightweight styles are changed by providing alternate resources with the same key. All Uno Material styles support the capability to be customized through resource overrides without the need to redefine the style.
+
+Overriding resources from Uno Material can be done at the app, page, or even control level.
+
+## Resizetizer for easy app images management
+
+[Resizetizer](xref:Uno.Resizetizer.GettingStarted) is a set of MSBuild tasks designed to manage an application's assets. With this package, there is no need to worry about creating and maintaining various image sizes or setting up a splash screen. Simply provide an SVG file, and the tool will handle everything else.
+
+## Support for SVG
+
+Today's apps run on such a wide range of displays, it is sometimes hard to make sure your icons always look crisp and sharp. To counter this, Uno Platform has a built-in support for the vector-based SVG format!
+
+## Progressive Web Apps (PWAs)
+
+Your Uno Platform WebAssembly app can run as a Progressive Web App. This means users can install it and run as a normal application on their device. In addition, this gives you even more customization options and additional PWA-specific APIs like badge notifications. You can install the [Nuget Package Explorer](https://nuget.info/) PWA like a local app!
+
+## WebView across platforms
+
+When you need to display web content within your cross-platform apps, you can use our `WebView2` control. Not only will it easily handle rendering of both remote and in-app HTML content, but it will also allow you to implement bi-directional JavaScript communication without any hassle. Note that `WebView2` is currently available on mobile and WinAppSDK targets. We are working on an implementation for Skia targets.
+
+## Media Player across platforms
+
+Uno Platform provides the `MediaPlayerElement` control which allows your app to play media files back in your app. It is supported on Linux through libVLC.
+
+## Monaco Editor on Wasm
+
+For all your in-app text and code editing needs you can use our port of the Monaco Editor – the same that is used by Visual Studio Code. With full support for syntax highlighting, you could even build your
+own WebAssembly IDE!
+
+## Migration path with WPF Uno Islands
```

Comment:
There was a previous section about Uno Islands. I think both sections should be merged into one.

Reply:
Done